df.tile-types.xml

```diff
@@ -4789,8 +4789,6 @@
             <item-attr name='special' value='TRACK'/>
             <item-attr name='direction' value='NSEW'/>
         </enum-item>
-        <enum-item/>
```

Comment:
as i said previously, if those are actually on toady's provided list, they should also be included here with an appropriate name